packages/manager/modules/web-paas/src/components/delete-user/controller.js

```diff
@@ -2,16 +2,31 @@ import get from 'lodash/get';
 
 export default class {
   /* @ngInject */
-  constructor($translate, WebPaas) {
+  constructor($translate, atInternet, WebPaas) {
     this.$translate = $translate;
+    this.atInternet = atInternet;
     this.webPaasService = WebPaas;
   }
 
   $onInit() {
     this.isLoading = false;
   }
 
+  cancel() {
+    this.atInternet.trackClick({
+      name:
+        'web-paas::project-user-license::user-table-options::delete-user-cancel',
```

Comment:
in some places tracking name is starting with "web-paas-platform-sh". Here it starts with 'web-paas'. Is it ok?

Reply:
The ones which are directing to platform sh links have  "web-paas-platform-sh" as tracking so its okay